docs/sources/enterprise/license-expiration.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "License Expiration"
+description = ""
+keywords = ["grafana", "licensing"]
+type = "docs"
+[menu.docs]
+parent = "enterprise"
+weight = 8
++++
+
+# Expired License
+
+If your license has expired, your Grafana admin needs to upload a new *license.jwt* file to the location in the `enterprise.license_path` directory (default your Grafana data directory). This file can be downloaded from your Grafana Cloud organization’s license page. Restart Grafana once the file is in place.
+
+Although you'll be able to access most of Grafana as usual, some functionality will stop working when the license expires and a banner will appear informing all users that the license has expired.
+
+> You should replace your license as soon as possible. Running Grafana Enterprise with an expired license is unsupported and can lead to unexpected consequences.
+
+## Datasource Permissions
+
+- Your current data source permissions will keep working as expected, but you'll be unable to add new data source permissions until the license has been replaced.
```

Comment:
Do you mean renewed rather than replaced?

Reply:
🤔 Probably, yes. Technically you don't renew a license but rather replace it but that's really not relevant.